go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/9elements/bmc-test-go
 
-go 1.23.1
+go 1.23
```

Comment:
Why changing this?

Reply:
the gopher said so https://github.com/9elements/bmc-test-go/actions/runs/10815516997/job/30004587016

Reply:
Fair enough